src/sections/schedule/index.jsx

```diff
@@ -152,6 +149,7 @@ function ScrollableTabsButtonAuto() {
       </AppBar>
       <TabPanel value={value} index={0}>
         <div className="scrollarrows_wrapper">
+          <h6 style={date}>Dec 7, 2020</h6>
```

Comment:
These changes for adding date doesn't seem like a good thing at the moment, as they feel abrupt and don't align with the look & feel of the web page.